cluster/addons/podsecuritypolicies/unprivileged.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: unprivileged
+  annotations:
+    kubernetes.io/description: 'unprivileged allows full access to all pod
+      settings except those that grant host privileges.'
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
+  labels:
+    kubernetes.io/cluster-service: 'true'
+    addonmanager.kubernetes.io/mode: Reconcile
+spec:
+  privileged: false
+  allowPrivilegeEscalation: true
```

Comment:
?

Reply:
+1 this seems like a mistake?

Reply:
`allowPrivilegeEscalation: false` doesn't do anything if the container is running as root.